src/App.js

```diff
@@ -27,7 +31,10 @@ function App() {
             );
             setUser(response.data);
         } catch (error) {
-            console.error(error);
+            setError((error) => ({
+                ...error,
```

Comment:
as there is only 1 property in error ('api'), no reason to spread here.
It will get overwritten on the next line, anyway.